lib/commands/deployproxy.js

```diff
@@ -718,9 +718,18 @@ function deployProxy(opts, request, done) {
       }
     }, function(err, req, body) {
       if (err) { return done(err); }
-
-      var jsonBody = (body ? JSON.parse(body) : null);
-
+    
+      var jsonBody;
+      try {
+        jsonBody = (body ? JSON.parse(body) : null);
+      } catch(e) {
+        // assume error if json parse fails
+        if (opts.verbose) { 
```

Comment:
Looking at the code below, if we omit logging here and don't return on line 731, the code will behave as it did before when encountering an error.  I think we should have an empty `catch` block and let the unparseable JSON fall through.  Thoughts?